src/BlockExplorerClient.ts

```diff
@@ -0,0 +1,12 @@
+import Axios from 'axios';
```

Comment:
Can you please move this file out from zksync.js to the client for now?
I think this is out of scope of this task.